### PR TITLE
[fmuobs] Set uniform loglevel for submodules

### PR DESCRIPTION
**Pull request by berland**
Not sure if this is the way to do it, but have not found a solution that automates this through interitance of the log level for already initialized loggers.

### CODE REVIEW

File: src/subscript/fmuobs/fmuobs.py [hunk 1]
```diff
@@ -312,9 +312,14 @@ def fmuobs(
...
         logger.setLevel(logging.INFO)
-
+        getLogger("subscript.fmuobs.parsers").setLevel(logging.INFO)
+        getLogger("subscript.fmuobs.writers").setLevel(logging.INFO)
+        getLogger("subscript.fmuobs.util").setLevel(logging.INFO)
     if debug:
         logger.setLevel(logging.DEBUG)
+        getLogger("subscript.fmuobs.parsers").setLevel(logging.DEBUG)
+        getLogger("subscript.fmuobs.writers").setLevel(logging.DEBUG)
+        getLogger("subscript.fmuobs.util").setLevel(logging.DEBUG)
```
Comment (jcrivenaes) [inline src/subscript/fmuobs/fmuobs.py:323]: I think you can make this shorter by avoiding repetition:

```
loglevel = logging.INFO
if debug:
     loglevel = logging.DEBUG

logger.setLevel(loglevel)
getLogger("subscript.fmuobs.parsers").setLevel(loglevel)
...
```

**Comment by jcrivenaes** [comment]
Not sure exactly what you want to achieve, but as idea(?) I have set fmu-dataio to inherit logging level by using a trick in the __init__.py file: https://github.com/equinor/fmu-dataio/blob/main/src/fmu/dataio/__init__.py

**Comment by berland** [comment]
> Not sure exactly what you want to achieve, but as idea(?) I have set fmu-dataio to inherit logging level by using a trick in the **init**.py file: https://github.com/equinor/fmu-dataio/blob/main/src/fmu/dataio/__init__.py

Part of the problem is that the loggers are already initialized when the `--verbose` or `--debug` command line options are parsed.

**Review by jcrivenaes — APPROVED** [review]
I think this is OK as it is just logging (messages) behaviour.

File: src/subscript/fmuobs/fmuobs.py (hunk 1 above)
Comment (berland) [reply src/subscript/fmuobs/fmuobs.py:323]: fixed

**Pull request merged**